src/components/two-up-display.sass

```diff
@@ -48,6 +48,9 @@
     overflow: hidden
     position: relative
 
+  .scrollable
+    overflow: auto
+
```

Comment:
@sfentress I wasn't sure if making the entire `content` class scrollable would break how you display the Populations model where overflow is hidden.  For now, I'm only adding the `scrollable` class to the right panel, but even that might need to be more nuanced (in case it clashes with how the graph is displayed.

Reply:
Yeah, we'll want to keep an eye on it, because of all the parts that expect to know their container's size, but it seems to work fine right now. If you add enough text to force the scroll bar to show, it does, and the graph still seems to size correctly. :+1: